hubs/templates/hubs.html

```diff
@@ -3,13 +3,15 @@
   <title>{{ hub.name }}</title>
   <link href="{{ url_for('static', filename='bootstrap/css/bootstrap.css') }}" rel="stylesheet" />
   <link href="{{ url_for('static', filename='css/style.css') }}" rel="stylesheet" />
+  <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,700,400italic|Comfortaa' rel='stylesheet' type='text/css'>
+<!--I added google fonts!!!! Sorry if I wasn't supposed to do that!!! -->
```

Comment:
Yup, we can use it for now but we need to pull it locally eventually.

(Fedora users don't like it if we send some of their traffic to google automatically when they use our websites).